src/main/java/cmd_utilities/CmdManager.java

```diff
@@ -0,0 +1,47 @@
+package cmd_utilities;
+
+import commands.*;
+
+import java.util.HashMap;
+
+public class CmdManager {
+    private final HashMap<String, Command> commandCollection = new HashMap<>();
+
+    public CmdManager() {
+    }
+
+    public void init() {
+        registerCommand("add", new Add());
```

Comment:
Command objects already have a `name` method, why are you writing the names again instead of calling the method?